src/main/webapp/app/pages/annotationPage/MutationEffectDescription.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { AlterationPageLink } from 'app/shared/utils/UrlUtils';
+import WithSeparator from 'react-with-separator';
+import SummaryWithRefs from 'app/oncokb-frontend-commons/src/components/SummaryWithRefs';
+
+const MutationEffectDescription: React.FunctionComponent<{
+  hugoSymbol: string;
+  description: string | undefined;
+}> = props => {
+  // Add link to alteration page when the allele is unknown but multiple alternative alleles at the same position have mutation effects
```

Comment:
Do we want to also add a link to alteration page when there is 1 AA?

Reply:
There would not be a case like that. When only one AA, we include the AA mutation description in the `description`